### ISSUE TITLE
App breaks when changing action type in unknown transaction

### ISSUE THREAD

**Issue by arrenv** [JoinColony/colonyCDapp#1839]
## Steps to reproduce

1. Perform an arbitrary transaction in a Colony.
2. This can be done by calling the `makeArbitraryTransaction` colony contract method directly.

You can see this in product in the Betacolony or by accessing this transaction directly - https://app.colony.io/beta?tx=0x5817c758c51c9200201ae4c4fa72fc11de19793cdbf018327b22b2e7a0661544

## Expected behaviour

* Opening an unknown transaction in the action panel and changing the "Choose action type" should not break the app.

## Actual behaviour

* Opening an unknown transaction in the action panel and changing the "Choose action type" breaks the app.

![image](https://github.com/JoinColony/colonyCDapp/assets/33682027/ffe1070e-d6a0-4715-8325-054c1a2e1c06)

![unknown-transaction](https://github.com/JoinColony/colonyCDapp/assets/33682027/cbb68ab9-8f91-4000-ac37-e4b21101fa37)

**Comment by rdig** [comment]
This title is kinda misleading :)

This issue doesn't need to "fix" the changing title interaction, but rather to deal with handling arbitrary transactions in the UI

**Comment by arrenv** [comment]
> This title is kinda misleading :)
> 
> This issue doesn't need to "fix" the changing title interaction, but rather to deal with handling arbitrary transactions in the UI

Yeah, I figured that was more of a feature that needs designs, contract updates, etc. I just wanted to make sure the app didn't break when opening up one of these transactions in the meantime.

**Comment by rdig** [comment]
Closing this until we re-spec it properly and re-prioritize it
